sensor/kubernetes/listener/resources/convert.go

```diff
@@ -59,6 +59,8 @@ type deploymentWrap struct {
 	original         interface{}
 	portConfigs      map[portRef]*storage.PortConfig
 	pods             []*v1.Pod
+	// TODO: we could have the networkPoliciesApplied stored here. This would require changes in the ProcessDeployment functions of the detector.
+	// networkPoliciesApplied augmentedobjs.NetworkPoliciesApplied
```

Comment:
What would be the advantages of having the `networkPoliciesApplied` stored here?

Reply:
If we decide to have it there, later in the `enricher` we only need to access this field. It's a matter of preference on where to put the code right now.

In the future, storing the `networkPoliciesApplied` (somewhere) will allow us to compare changes between how the deployment was affected and how it is affected now.